sessionctx/variable/varsutil_test.go

```diff
@@ -605,17 +597,20 @@ func (s *testVarsutilSuite) TestValidate(c *C) {
 		{TiDBAllowFallbackToTiKV, "tiflash,tikv,tidb", true},
 	}
 
-	for _, t := range tests {
-		_, err := GetSysVar(t.key).Validate(v, t.value, ScopeGlobal)
-		if t.error {
-			c.Assert(err, NotNil, Commentf("%v got err=%v", t, err))
-		} else {
-			c.Assert(err, IsNil, Commentf("%v got err=%v", t, err))
-		}
+	for _, tc := range testCases {
+		t.Run(tc.key, func(t *testing.T) {
+			t.Parallel()
+			_, err := GetSysVar(tc.key).Validate(v, tc.value, ScopeGlobal)
```

Comment:
PTAL @tisonkun 
After I saw the comment - https://github.com/pingcap/tidb/pull/27848#discussion_r707003622 I decided to review the changes I had made with this PR once more.

Is it likely that running the `TestValidate` function may cause any `fatal error`? 
Since `func GetSysVar()` and `func Validate()` do not write to any `map` so I think it is safe to run these test cases in parallel.

Please confirm.

Reply:
Yes. It doesn't hurt I think.

Reply:
PTAL @tisonkun 
I came across the issue ref- #27779 about copying the iterator variable into a new variable for running tests in parallel. Should I make a fix to this file? If yes, since this PR is already merged, should I create a new issue and then make the changes. Please suggest.

Reply:
@penthaapatel `GetSysVar` will be run in the server concurrently during regular execution. `Validate()` is never run concurrently with the same `v` (`SessionVars`), but because it is not expected to modify data (only validate) it should not concurrent write to any maps.

Reply:
@morgo Thank you for clearing that up.